bids-validator/validators/json/schemas/microscopy.json

```
@@ -0,0 +1,78 @@
+{
+  "type": "object",
+  "properties": {
+    "Manufacturer": { "type": "string", "minLength": 1 },
+    "ManufacturersModelName": { "type": "string", "minLength": 1 },
+    "DeviceSerialNumber": {"type": "string", "minLength": 1},
+    "StationName": { "type": "string", "minLength": 1 },
+    "SoftwareVersions": { "type": "string", "minLength": 1 },
+    "InstitutionName": { "type": "string", "minLength": 1 },
+    "InstitutionAddress": { "type": "string", "minLength": 1 },  
+    "InstitutionalDepartmentName": { "type": "string", "minLength": 1 }, 
+    "BodyPart": { "type": "string", "minLength": 1 },
+    "BodyPartDetails": { "type": "string", "minLength": 1 },
+    "BodyPartDetailsOntology": { "type": "string", "minLength": 1, "format": "uri" },
+    "SampleEnvironment": { "type": "string", "minLength": 1, "enum": ["in vivo", "ex vivo", "in vitro"]  },
+    "SampleEmbedding": { "type": "string", "minLength": 1 },
+    "SampleFixation": { "type": "string", "minLength": 1 },
+    "SampleStaining": { 
+      "anyOf": [
+        { "type": "string", "minLength": 1 },
+        { "type": "array", "items": { "type": "string" } }
+      ]
+    },
+    "SamplePrimaryAntibody": { 
+      "anyOf": [
+        { "type": "string", "minLength": 1 },
+        { "type": "array", "items": { "type": "string" } }
+      ]
+    },
+    "SampleSecondaryAntibody": { 
+      "anyOf": [
+        { "type": "string", "minLength": 1 },
+        { "type": "array", "items": { "type": "string" } }
+      ]
+    },
+    "SliceThickness": { "type": "number" },
+    "SampleExtractionProtocol": { "type": "string", "minLength": 1 },
+    "SampleExtractionInstitution": { "type": "string", "minLength": 1 },
+    "TissueDeformationScaling": { "type": "number" },
+    "PixelSize": {"type": "array", "items": { "type": "number" } },
+    "PixelSizeUnits": { "type": "string", "enum": ["mm", "um", "nm"] },
+    "Immersion": { "type": "string", "minLength": 1 },
+    "NumericalAperture": { "type": "number" },
+    "Magnification": { "type": "number" },
+    "ImageAcquisitionProtocol": { "type": "string", "minLength": 1 },
+    "OtherAcquisitionParameters": { "type": "string", "minLength": 1 },
+    "ChunkTransformationMatrix": {"type": "array", "items": { "type": "array", "items":{ "type": "number"} } },
```

```suggestion
    "ChunkTransformationMatrix": {
      "anyOf": [
        {"type": "array", "minItems": 3, "maxItems": 3, "items": { "type": "array", "minItems": 3, "maxItems": 3, "items":{ "type": "number"} } },
        {"type": "array", "minItems": 4, "maxItems": 4, "items": { "type": "array", "minItems": 4, "maxItems": 4, "items":{ "type": "number"} } }
      ]
    },
```
To make sure the matrix is either 3x3 or 4x4.